ateam-common-packets/include/stspin_c_static_checks.h

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file stspin_c_static_checks.h
+ * @author Will Stuckey
+ * @brief contains the static checks seemingly incompatible with Rust bindgen
```

Comment:
Pretty sure you just need to add `#include <assert.h>` to get it to work

Reply:
Ah I thought this was a first class feature. Ill see if that fixes it